chrome-ext/newtab.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>EXT</title>
+</head>
+
+<body>
+    <main>
+        <P></P>
```

Comment:
Curious to know what a upper case P was used here instead of the standard practice lower case p?

Reply:
no reason at all.. just rushing :P